apps/transmitmedia.cpp

```diff
@@ -582,36 +582,36 @@ void SrtCommon::ConnectClient(string host, int port)
     if ( stat == SRT_ERROR )
     {
         srt_close(m_sock);
-        Error(UDT::getlasterror(), "UDT::connect");
+        Error("srt_connect");
     }
 
     stat = ConfigurePost(m_sock);
     if ( stat == SRT_ERROR )
-        Error(UDT::getlasterror(), "ConfigurePost");
+        Error("ConfigurePost");
 }
 
-void SrtCommon::Error(UDT::ERRORINFO& udtError, string src)
+void SrtCommon::Error(string src)
 {
-    int udtResult = udtError.getErrorCode();
-    string message = udtError.getErrorMessage();
-    Verb() << "\nERROR #" << udtResult << ": " << message;
+    int errnov = 0;
+    int result = srt_getlasterror(&errnov);
+    string message = srt_getlasterror_str();
+    Verb() << "\nERROR #" << result << "." << errnov << ": " << message;
 
-    udtError.clear();
     throw TransmissionError("error: " + src + ": " + message);
```

Comment:
`clear()` usage deleted - OK

Reply:
Not completely. See `srt_clearlasterror()`. Not sure if this is something extremely required, but changes in the API require deprecation period.

Reply:
Might be more correct to delete m_TLSError, instead of reseting it via `CUDTUnited::getError().clear()` in `srt_clearlasterror()`.
But better to resolve this separately.

```
CUDTException* CUDTUnited::getError()
{
    if(!pthread_getspecific(m_TLSError))
        pthread_setspecific(m_TLSError, new CUDTException);
    return (CUDTException*)pthread_getspecific(m_TLSError);
}
```